build.gradle

```diff
@@ -35,6 +35,7 @@ dependencies {
     implementation 'net.william278:Annotaml:2.0.1'
     implementation 'dev.dejvokep:boosted-yaml:1.3.1'
     implementation 'de.themoep:minedown-adventure:1.7.2-SNAPSHOT'
+    implementation 'net.william278:DesertWell:1.1.1'
```

Comment:
We'll need to ensure this gets relocated in the jarfile, too, since we're implementing it.
```groovy
relocate 'net.william278.desertwell', 'net.william278.velocitab.libraries.desertwell'
```